spec/commands/config_spec.rb

```diff
@@ -14,9 +14,12 @@
       bundle "config path vendor/bundle"
       bundle "install"
 
-      expect(bundled_app(".bundle")).not_to exist
-      expect(tmp("foo/bar/config")).to exist
-      should_be_installed "rack 1.0.0"
+      # See cli/install.rb:L86
```

Comment:
please don't lint to lines -- use methods, which are less likely to go out of date